graders/r/r-requirements.R

```diff
@@ -16,13 +16,16 @@ if(Ncpus > 4) {
 # during package installation/updation and set the default mirror
 options(Ncpus = Ncpus, repos = c("CRAN" = "https://cran.rstudio.com"))
 
-# The following are packages used in STAT 385
-pkg_list = c('RcppArmadillo')
+# The following are packages used in STAT 432
+pkg_list = c('randomForest', 'caret')
 
 # Determine what packages are NOT installed already.
 to_install_pkgs = pkg_list[!(pkg_list %in% installed.packages()[,"Package"])]
 
 # Install the missing packages
 if(length(to_install_pkgs)) {
-    install.packages(to_install_pkgs)
+    install.packages(to_install_pkgs, quiet = TRUE, verbose = FALSE)
 }
+
+# Install pl testing framework package from GitHub
+remotes::install_github('illinois-r/pltest')
```

Comment:
This allows us to easily update the underlying autograding code as needed.

Previously, each course had a copy of the code under `pltest.R` within `serverFilesCourse`.